src/Tgstation.Server.Host/Models/Migrations/20180906135553_MSInitialCreate.cs

```diff
@@ -290,7 +290,8 @@ protected override void Up(MigrationBuilder migrationBuilder)
 					AccessToken = table.Column<string>(nullable: true),
 					PushTestMergeCommits = table.Column<bool>(nullable: false),
 					ShowTestMergeCommitters = table.Column<bool>(nullable: false),
-					AutoUpdatesKeepTestMerges = table.Column<bool>(nullable: false),
+                    PostTestMergeComment = table.Column<bool>(nullable: false),
```

Comment:
You never, ever, EVER modify a migration

First do your research on the concept: https://en.wikipedia.org/wiki/Schema_migration

Then work on properly adding a new one: https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/